diff -Naur Webware-0.9.4-original/WebKit/HTTPResponse.py Webware-0.9.4-patched/WebKit/HTTPResponse.py
--- Webware-0.9.4-original/WebKit/HTTPResponse.py	2007-05-17 14:42:56.000000000 -0700
+++ Webware-0.9.4-patched/WebKit/HTTPResponse.py	2009-05-14 18:33:11.002963000 -0700
@@ -345,7 +345,7 @@
 		app = trans.application()
 		if not app.setting('UseCookieSessions'):
 			return
-		sess = trans.session()
+		sess = trans._session
 		if sess:
 			cookie = Cookie(app.sessionName(trans), sess.identifier())
 			cookie.setPath(app.sessionCookiePath(trans))
