--- Webware-0.9.4/WebKit/ServletFactory.py	(revision 7115)
+++ Webware-0.9.4/WebKit/ServletFactory.py	(revision 7116)
@@ -182,8 +182,11 @@
 			# check if __init__.py is in the directory
 			packageDir = os.path.join(directory, moduleName)
 			initPy = os.path.join(packageDir, '__init__.py')
-			if not os.path.exists(initPy):
-				# if it does not exist, make an empty one
+			for ext in ('', 'c', 'o'):
+				# the __init__.py file may only exist in compiled form
+				if os.path.exists(initPy + ext):
+					break
+			else: # if it does not exist, make an empty one
 				file = open(initPy, 'w')
 				file.write('#')
 				file.close()
